netmap/selector_test.go

```diff
@@ -296,6 +473,7 @@ func TestPlacementPolicy_ProcessSelectors(t *testing.T) {
 }
 
 func TestPlacementPolicy_ProcessSelectorsHRW(t *testing.T) {
+	t.Skip()
```

Comment:
what is this for?

Reply:
I added it while making some changes, will remove.